### PR TITLE
[BEAM-5435] Kryo separated from euphoria-core to extensions-kryo.

### PR DESCRIPTION
**Pull request by VaclavPlajt**
Separation of Kryo.
------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

**Comment by dmvk** [comment]
I think we should move this to beam-sdks-java-extensions-kryo and make it completely separated from euphoria dsl.

File: sdks/java/extensions/euphoria/euphoria-kryo/build.gradle [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+description = "Apache Beam :: SDKs :: Java :: Extensions :: Euphoria Java 8 DSL :: Kryo"
+
+ext {
+    kryoVersion = '4.0.2'
+}
+
+dependencies {
+    shadow project(path: ":beam-sdks-java-core", configuration: "shadow")
+    shadow "com.esotericsoftware:kryo:${kryoVersion}"
+    //testCompile library.java.slf4j_api
```
Comment (dmvk) [inline sdks/java/extensions/euphoria/euphoria-kryo/build.gradle]: can we remove this?

**Review by dmvk — CHANGES_REQUESTED** [review]

File: sdks/java/extensions/euphoria/euphoria-kryo/build.gradle (hunk 1 above)
Comment (VaclavPlajt) [reply sdks/java/extensions/euphoria/euphoria-kryo/build.gradle]: sure

**Comment by VaclavPlajt** [comment]
Kryo integration moved to `beam-sdks-java-extensions-kryo`.

**Pull request merged**